modules/monitoring-resources-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -0,0 +1,155 @@
+
+=== openshift-monitoring/node-exporter
+
+Expose the `/metrics` endpoint on port 9100. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/openshift-state-metrics
+
+Expose openshift-state-metrics `/metrics` endpoints within the cluster on the following ports:
+
+* Port 8443 provides access to the OpenShift resource metrics. This port is for internal use, and no other usage is guaranteed.
+* Port 9443 provides access to the internal `openshift-state-metrics` metrics. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/prometheus-k8s
+
+Expose the Prometheus web server within the cluster on the following ports:
+
+* Port 9091 provides access to all the Prometheus endpoints. Granting access requires binding a user to the `cluster-monitoring-view` cluster role.
+
+* Port 9092 provides access to the `/metrics` and `/federate` endpoints only. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-user-workload-monitoring/prometheus-operator
+
+Expose the `/metrics` endpoint on port 8443. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/prometheus-operator
+
+Expose the `/metrics` endpoint on port 8443. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-user-workload-monitoring/prometheus-user-workload
+
+Expose the Prometheus web server within the cluster on the following ports:
+
+* Port 9091 provides access to the `/metrics` endpoint only. This port is for internal use, and no other usage is guaranteed.
+* Port 9092 provides access to the `/federate` endpoint only. Granting access requires binding a user to the `cluster-monitoring-view` cluster role.
+
+This also exposes the `/metrics` endpoint of the Thanos sidecar web server on port 10902. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/telemeter-client
+
+Expose the `/metrics` endpoint on port 8443. This port is for internal use, and no other usage is guaranteed.
+
+=== openshift-monitoring/thanos-querier
+
+Expose the Thanos Querier web server within the cluster on the following ports:
+
+* Port 9091 provides access to all the Thanos Querier endpoints. Granting access requires binding a user to the `cluster-monitoring-view` cluster role.
+* Port 9092 provides access to the `/api/v1/query`, `/api/v1/query_range/`, `/api/v1/labels`, `/api/v1/label/*/values`, and `/api/v1/series` endpoints restricted to a given project. Granting access requires binding a user to the `view` cluster role in the project.
+* Port 9093 provides access to the `/api/v1/alerts`, and `/api/v1/rules` endpoints restricted to a given project. Granting access requires binding a user to the `monitoring-rules-edit` cluster role or `monitoring-edit` cluster role or `monitoring-rules-view` cluster role in the project.
```

Comment:
```suggestion
* Port 9093 provides access to the `/api/v1/alerts`, and `/api/v1/rules` endpoints restricted to a given project. Granting access requires binding a user to the `monitoring-rules-edit` cluster role, `monitoring-edit` cluster role, or `monitoring-rules-view` cluster role in the project.
```

Reply:
I'll go ahead and fix it here, and write up a ticket to fix it in the source, too. Thanks!